code/game/machinery/harvester.dm

```diff
@@ -91,8 +91,18 @@
 /obj/machinery/harvester/proc/start_harvest()
 	if(!occupant || !iscarbon(occupant))
 		return
-	var/mob/living/carbon/C = occupant
-	operation_order = reverseList(C.bodyparts)   //Chest and head are first in bodyparts, so we invert it to make them suffer more
+
+	var/mob/living/carbon/carbon_occupant = occupant
+
+	if(carbon_occupant.stat < UNCONSCIOUS)
```

Comment:
Can’t you just check for being alive, like an isalive() thing? Using defines like integers makes me feel weird

Reply:
I'm not aware of any isalive helpers and stat defines are used like this all over the code. I know it's kind of janky but as a far as I know it's common practice for stat stuff.

It might be possible to create a one-size-fits-all helper for liveliness checks but I feel that would be out of scope. Maybe in a follow up PR?

Reply:
Nice safe

Reply:
But yeah seems fine for now then